packages/testsuite/cypress/e2e/runtime/test-runtime-open-ports.cy.ts

```diff
@@ -0,0 +1,59 @@
+describe("TESTS: Runtime => Server => Open Ports", () => {
+  let managementEndpoint: string;
+  let serverHost: string;
+
+  before(() => {
+    cy.startWildflyContainer().then((result) => {
+      managementEndpoint = result as string;
+      cy.task("execute:cli", {
+        managementApi: managementEndpoint + "/management",
+        operation: "read-attribute",
+        address: [],
+        name: "name",
+      }).then((result) => {
+        serverHost = (result as { result: string }).result;
+      });
+    });
+  });
+
+  after(() => {
+    cy.task("stop:containers");
+  });
+
+  it("Read open ports", () => {
+    cy.navigateTo(managementEndpoint, "runtime");
+    cy.get(`#standalone-host-${serverHost}`).click();
+    cy.get("#hal-finder-preview").should("be.visible");
+    cy.get("#h2-open-ports").should("be.visible");
+    cy.task("execute:cli", {
```

Comment:
Can you use `getDefaultBooleanValue`method? For sub-attributes `socket-binding-group` and `socket-binding` you can use `bound-port.socket-binding-group` or `bound-port.socket-binding` for name parameter.

Reply:
That call doesn't retrieve boolean values, translation of that call would be `/socket-binding-group=standard-sockets/socket-binding=*:read-attribute(name=bound-port)`, which returns a list of objects, I don't think this is a right place to use `getDefaultBooleanValue` method.

Reply:
What do you think about new method to get a string value or list?  implementation could be very similar to `getDefaultBooleanValue`

Reply:
Yea It sounds better. :1st_place_medal:

Reply:
Can I resolve this one then? 🙂